src/feature/payment-history/localeTextConstants.jsx

```diff
@@ -0,0 +1,135 @@
+const noNB = {
```

Comment:
Could be cleaner to move this file to a new separate 'locales' directory at root level as it could be a shared asset for all datagrids

Reply:
Moved grid translations to a new folder under common.